frontend/lib/ui/style/variables.css

```diff
@@ -118,48 +111,48 @@
   --base-black: #000000;
   --base-brand-blue: #113051;
   --base-dark-blue: #00305b;
-  --base-light-grey: #f0f0f0;
-  --base-red: #e3032e;
   --base-white: #ffffff;
 
-  --accent-blue: #5d81b8;
-  --accent-lime: #dddb00;
-  --accent-orange: #e87c59;
-  --accent-light-blue: #c9d8e9;
-  --accent-light-lime: #dddda6;
-  --accent-light-orange: #e9dace;
-  --accent-dark-blue: #00305b;
-  --accent-dark-green: #3d6315;
-  --accent-dark-red: #8f131e;
-  --accent-dark-grey: #878787;
-
-  --blue-light-100: #e0f2fe;
-  --blue-light-800: #065986;
-  --menu-updates: #004eeb;
-  --link-default: #155eef;
+  --accent-orange: #ef8050;
+
+  --link-default: var(--border-color-focus);
   --link-hover: #004eeb;
   --link-active: #0040c1;
   --link-visited: #6927da;
```

Comment:
@jorisleker Do you have the colours for link-hover, link-active and link-visited on Figma somewhere and have these been updated also?

Reply:
@Lionqueen94 yes, they are on the [Fundamentals page](https://www.figma.com/design/zZlFr8tYiRyp4I26sh6eqp/Kiesraad---Abacus-optelsoftware?node-id=3087-58933&node-type=frame&t=nOtDRFnLZ9hw23eb-11). Not sure whether they were changed. I think not, but please verify!